src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/reporting/ndmis/performance/NdmisPerformanceReportJobConfiguration.kt

```diff
@@ -161,16 +168,30 @@ class NdmisPerformanceReportJobConfiguration(
       .reader(ndmisReader)
       .processor(processor)
       .writer(writer)
+      .faultTolerant()
+      .skipPolicy(skipPolicy)
       .build()
   }
 
   private val pushToS3Step = stepBuilderFactory["pushToS3Step"]
     .tasklet { _: StepContribution, _: ChunkContext ->
       listOf(referralReportPath, complexityReportPath, appointmentsReportPath).forEach { path ->
         s3Service.publishFileToS3(ndmisS3Bucket, path, "export/csv/reports/")
-        File(path.toString()).delete()
+//        File(path.toString()).delete()
```

Comment:
what's the reason for commenting this line out?

Reply:
accident. (used to test the reports locally)